packages/style-engine/src/styles/index.ts

```diff
@@ -2,7 +2,13 @@
  * Internal dependencies
  */
 import color from './color';
+import elements from './elements';
 import spacing from './spacing';
 import typography from './typography';
 
-export const styleDefinitions = [ ...color, ...spacing, ...typography ];
+export const styleDefinitions = [
+	...color,
+	...elements,
```

Comment:
TODO:

This effectively means that we'll be returning elements styles along with block styles for color, typography et. al., which is not a real world use case. 

At least inline style we'd want these to be separate.

Reply:
Actually, I think it's better to bring the frontend to the same status as the backend (https://github.com/WordPress/gutenberg/pull/40987), the worry about whether and how we deal with entire elements objects later.